CHANGES.rst

```diff
@@ -22,6 +22,8 @@ API Changes
 Bug Fixes
 ---------
 
+- Box and xrange zoom tools. [#997]
```

Comment:
Are you sure this is not a new feature?

Reply:
woops, good catch.  Should this go at the top of "New Features" (since it is applicable to multiple configurations/viewers) or in "Other Changes and Additions"?

Reply:
moved to "Other Changes and Additions" - if you think it belongs better elsewhere, let me know!

Reply:
IMHO "new features" is better suited because this is a nice cool new feature but I'll leave it to your good judgement. Either way, since it affects multiple layouts, it can be outside the subsections.